client/src/modules/break_even_reference/modals/break_even_reference.modal.html

```diff
@@ -0,0 +1,137 @@
+<form name="BreakEvenReferenceForm" bh-submit="BreakEvenReferenceModalCtrl.submit(BreakEvenReferenceForm)" novalidate>
+  <div class="modal-header">
+    <ol class="headercrumb">
+      <li class="static" translate>TREE.ADMIN</li>
+      <li class="title">
+        <span translate>TREE.BREAK_EVEN_REFERENCE</span>
+        <label ng-if="BreakEvenReferenceModalCtrl.isCreating" class="badge badge-warning" translate>FORM.LABELS.CREATE</label>
+        <label ng-if="!BreakEvenReferenceModalCtrl.isCreating" class="badge badge-warning" translate>FORM.LABELS.UPDATE</label>
+      </li>
+    </ol>
+  </div>
+
+  <div class="modal-body" style="overflow-y: scroll; height:300px;">
+    <!-- label -->
+    <div class="form-group" ng-class="{ 'has-error' : BreakEvenReferenceForm.$submitted && BreakEvenReferenceForm.label.$invalid }">
+      <label class="control-label" translate>FORM.LABELS.DESIGNATION</label>
+      <div>
+        <input name="label" ng-model="BreakEvenReferenceModalCtrl.reference.label" autocomplete="off" ng-maxlength="90" class="form-control" required>
+        <div class="help-block" ng-messages="BreakEvenReferenceForm.label.$error" ng-show="BreakEvenReferenceForm.$submitted">
+          <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+        </div>
+      </div>
+    </div>
+
+    <!-- account_reference_id -->
+    <bh-account-reference-select
+      id="account_reference_id"
+      account-reference-id="BreakEvenReferenceModalCtrl.reference.account_reference_id"
+      label="FORM.SELECT.ACCOUNT_REFERENCE"
+      on-select-callback="BreakEvenReferenceModalCtrl.onSelectAccountReference(accountReference)"
+      required="true">
+    </bh-account-reference-select>
+
+    <!-- is_cost -->
+    <div class="form-group" ng-class="{'has-error' : BreakEvenReferenceForm.$submitted && BreakEvenReferenceForm.is_cost.$invalid }">
+      <label translate>FORM.LABELS.TYPE</label>
+      <div class="radio">
+        <label class="radio-inline">
+          <input
+            type="radio"
+            name="is_cost"
+            ng-value="1"
+            ng-click="BreakEvenReferenceModalCtrl.inSetRevenueType()"
```

Comment:
Just FYI - in English, we either use "unset" or "reset" to indicate the opposite of "set" instead of "inset", which means to insert something.